README.rst

```diff
@@ -64,14 +70,29 @@ Strawberry Fields requires Python version 3.6, 3.7, or 3.8. Installation of Stra
 Getting started
 ===============
 
-To see Strawberry Fields in action immediately, try out our `Strawberry Fields Interactive <https://strawberryfields.ai>`_ web application. Prepare your initial states, drag and drop gates, and watch your simulation run in real time right in your web browser.
+To see Strawberry Fields in action immediately, try out our `Strawberry Fields Interactive
+<https://strawberryfields.ai>`_ web application. Prepare your initial states, drag and drop gates,
+and watch your simulation run in real time right in your web browser.
```

Comment:
Are you going to keep the progress bar? :)

Reply:
For now!